api/controllers/employees.js

```diff
@@ -0,0 +1,86 @@
+const sequelize = require('sequelize');
+
+const logger = require('../logger');
+const Employee = require('../models/employee');
+const Transaction = require('../models/transaction');
+
+const EmployeeController = {};
```

Comment:
😱 why aren't we using our great `class`es?

Reply:
I think it's not a case to use classes here, because we need just store some functions together in one space.
I think this one is more good way to change ("exports" instead of object with module exports)
https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes